enterprise/frontend/src/embedding-sdk/lib/plugins.ts

```diff
@@ -14,16 +14,16 @@ export type SdkClickActionPluginsConfig = (
   clickedDataPoint: SdkDataPointObject,
 ) => ClickAction[];
 
-type DashCardMenuCustomElement = ({
+export type DashCardMenuCustomElement = ({
   question,
 }: {
-  question: Question;
+  question: QuestionType;
 }) => ReactNode;
 
-type CustomDashCardMenuItem = ({
+export type CustomDashCardMenuItem = ({
   question,
 }: {
-  question?: Question;
+  question?: QuestionType;
 }) => DashCardMenuItem;
```

Comment:
If importing the types from `embedding-sdk/lib/plugins` does not work, we could try to re-export this to `embedding-sdk` entry point as well. Can you check if the. `QuestionType` parameter is resolved after building the SDK in release mode?